src/MainWindow.vala

```diff
@@ -78,6 +78,11 @@ public class Installer.MainWindow : Hdy.Window {
         add (overlay);
 
         language_view.next_step.connect (() => {
+            // Don't prompt for screen reader if we're able to navigate without it
+            if (orca_timeout_id != 0) {
+                Source.remove (orca_timeout_id);
```

Comment:
This remove call seems to kick a critical error to output for the ID not existing if you wait for the timeout to occur. I checked and the ID is definitely non-zero so I am not sure what is happening there, or if it is something we should be concerned about.

Reply:
I think I fixed it. Thanks for the attention to detail!

Reply:
Huh. Interesting that assigning the timeout ID to Source.REMOVE isn't taking care of that.

Reply:
But that fixed it!